### ISSUE TITLE
Avatar hand animation speed is too quick after changes to default locomotion configuration

### ISSUE THREAD

**Issue by paradoxical-autumn** [Yellow-Dog-Man/Resonite-Issues#3169]
### Describe the bug?

Currently, the hand animations on the default locomotion configuration while idle are too fast, causing an uncanny/unrealistic amount of hand movement. (I am unsure of the configuration values that are causing this, as I haven't dived into the component)

### To Reproduce

Spawn out an avatar without any locomotion animation configuration overrides, such as the default avatar.

### Expected behavior

For the default configuration to animate at a realistic pace.

### Screenshots

https://github.com/user-attachments/assets/17991ab0-5067-40e8-ab0d-7ff0f4d00e42

### Resonite Version Number

Beta 2024.11.12.1329

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

[DESKTOP-2IRVQ73 - 2024.11.12.1329 - 2024-11-12 22_46_51.log](https://github.com/user-attachments/files/17723673/DESKTOP-2IRVQ73.-.2024.11.12.1329.-.2024-11-12.22_46_51.log)

### Additional Context

As far as I'm aware, this is only appears in desktop mode.

_I am unsure if this is counts as an issue, but judging by the uncanny movement I created this anyway. It might serve as a disambiguation if this is intended behaviour._

### Reporters

- bobool3ol

**Comment by Frooxius** [comment]
I'm not sure if this is intentional or a wrong value got in, but it could be tweaked either way.

@BlueCyro would you have more insight on this?

**Comment by BlueCyro** [comment]
> I'm not sure if this is intentional or a wrong value got in, but it could be tweaked either way.
> 
> [@BlueCyro](https://github.com/BlueCyro) would you have more insight on this?

Afaik this is intentional, but I see no reason I can't tweak it down.

**Comment by troyBORG** [comment]
yeah I had my own locomotion setting and I think my fingers are even moving way faster

**Comment by troyBORG** [comment]
> I can't tweak it down.

![Image](https://github.com/user-attachments/assets/92bfefae-adae-4e0d-9729-1245217348c5)

This thing here that I'm driving is what needs to be adjusted probally..

I have mine set to go slower as my hand gets closer to my face...

**Comment by shiftyscales** [comment]
Thank you @troyBORG - but your quote is a bit out of context / misinterpreted. Cyro was saying that there are no blockers preventing that value from being tweaked / lessened. Hence:

> I see no reason I can't tweak it down.

**Comment by SlyTheFloof** [comment]
im quite happy with the hand movements now
